.github/workflows/build_centos7.yml

```diff
@@ -183,16 +183,6 @@ jobs:
           path: ${{ env.VCPKG_CACHE_DIR }}
           key: vcpkg-cache-centOS-${{ hashFiles('src/vcpkg.json', '.git/modules/vcpkg/HEAD') }}
 
-      - name: Running unit tests
-        timeout-minutes: 120
-        shell: bash
-        run: |
-          export PATH=${GITHUB_WORKSPACE}/_build/vcpkg_installed/x64-linux/tools/openmpi/bin/:$PATH
-          export LD_LIBRARY_PATH=LD_LIBRARY_PATH:${{ env.XPRS_LIB_Path_CONTAINER }}
-          export XPRESS=${{ env.XPRESS_CONTAINER }}
```

Comment:
How come the new scripts don't source XPRESS ?

Reply:
Mistake on my side. I've forgotten to remove this deletion. It was to disable test run during development